Sources/Plot/API/HTMLComponents.swift

```diff
@@ -579,6 +579,96 @@ extension List: ComponentContainer where Items == ComponentGroup {
     }
 }
 
+// Component used to render an `<picture>` element for responsive image support.
+public struct Picture: Component {
+    /// Type used to define an  Picture source, which points to an image file (or set of them) and associated media queries (and other paramaters).
+    public struct Source {
+        /// image or images  to use - can have parameters
+        public var srcset: String
+        /// sizes for the `srcset` to use - can have parameters
+        public var sizes: String?
+        ///  Media querry (optional)
+        public var media: String?
+        ///  Media type (optional)
+        public var type: String?
+        ///  Intrinsic height of image in pixels - integer with no unit (optional)
+        public var height: Int?
+        ///  Intrinsic width of image in pixels - integer with no unit (optional)
+        public var width: Int?
+
+        /// Initialize a new source.
+        /// - parameter srcset: The set of sources to which this element should point.
+        /// - parameter sizes: Comma separated list of media queries for the `srcset` images.
+        /// - parameter media: Media query for the resources intended media.
+        /// - parameter type: MIME type of the image, with optional `codecs` paramater.
+        /// - parameter height: Intrinsic height of image in pixels - integer with no unit.
+        /// - parameter width:  Intrinsic width of image in pixels - integer with no unit.
+
+        internal init(srcset: String, sizes: String? = nil, media: String? = nil, type: String? = nil, height: Int? = nil, width: Int? = nil) {
```

Comment:
Is this explicit initializer needed? Memberwise initializers are generated by the Swift compiler by default, so it seems like this can be removed.

Reply:
I am sure there is a great reason why we did this, but it is lost to the sands of time. :-) I think I generated it when I had code that did some other things there, but as the code got updated, it eventually became irrelevant. Now removed for your merging pleasure. :-)